### PR TITLE
manifest: pull zephyr with increased Thread buffers

### PR DESCRIPTION
**Pull request by MarekPorwisz**
Zephyr with increased child count and IP count per child.
Jira: https://projecttools.nordicsemi.no/jira/browse/KRKNWK-7796

- [x] https://github.com/nrfconnect/sdk-zephyr/pull/375
- [x] https://github.com/nrfconnect/sdk-nrfxlib/pull/306

Jira:
https://projecttools.nordicsemi.no/jira/browse/KRKNWK-7796

Signed-off-by: Marek Porwisz <marek.porwisz@nordicsemi.no>

### CODE REVIEW

**Review by edmont — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 33e59d44afb636c4fea4671a7ad301cbb619b7ec
+      revision: pull/304/head
```
Comment (pszkotak) [inline west.yml:104]: As https://github.com/nrfconnect/sdk-nrfxlib/pull/306 has been created I think that we should go with it here.

```suggestion
      revision: pull/306/head
```

**Review by greg-fer — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Comment by tejlmand** [comment]
Please rebase, and update zephyr-sdk SHA: 41ce8e165a9af4171ef52a73aede89da6ce60920

**Comment by tejlmand** [comment]
nrfxlib SHA: c481f548517367613f64e1eacb36792ed1a2e1a9

**Review by rlubos — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
sha checked

**Comment by MarekPorwisz** [comment]
The PR seems to be failing on on CI issues like git cloning or not starting some process over and over again...

**Comment by edmont** [comment]
CI is green now.

**Pull request merged**